Readme.md

```diff
@@ -81,3 +96,8 @@ To do a manual publish, please follow these steps:
 
 This will take about an hour before the update is available.
 
+## About this repository
+
+The `master` branch of this repository contains the VS Code extension for Prisma schema files. Prisma package dependencies are kept up to date with a GitHub Action workflow, that updates them every time a new version of them is released.
+
+There is a stable version `prisma` and an unstable version `prisma-dev`.
```

Comment:
LGTM, however, the code in `master` branch is from the unstable channel and the badges point to a stable channel. 

We should stick to one of stable or unstable in `master`.  

Related issue https://github.com/prisma/vscode/issues/119

Reply:
Isn't it that the stable channel does not exist in code any more really, and tags do exist on `master` that represent the stable channel, and there is no real difference between stable and unstable code?

Then I think keeping the badges is absolutely correct and what we want to achieve.